### ISSUE TITLE
Can I set up my own 'Bullets'

### ISSUE THREAD

**Issue by h1sashin** [rcaferati/react-awesome-slider#156]
As in title, can I use my own 'bullets' instead of using these build in? I tried something like `ref.current.currentIndex` but ony function that works for me was `ref.current.nextSlide()`. I haven't spotted anything in about it in documentation.

**Comment by upta** [comment]
I used to have a custom-built bullet solution, but it always had quirky issues if you'd click while the slider was animating and all-in-all it wasn't a very nice experience.

Recently I gave up on the custom bullets and went the route of re-styling the existing bullet markup and it went _much_ better and I was able to recreate the effect I'd had previously (similar to the small preview boxes under the slider you see in the Steam store).  It's still a little quirky, but definitely behaves better than it did before.

The awssld__bullets class (and his descendants) are what you're looking for if you want to go the restyle route.

**Comment by h1sashin** [comment]
Nah, I will try that, but styling built in bullets is not a solution I'm looking for. It's propably unstylable until they're children of AwesomeSlider component, and these bullets should do more than just changing current slide. Simple example below.
![Web 1920 – 1](https://user-images.githubusercontent.com/25825628/101146322-25230100-361b-11eb-8a90-77e282b23d6f.png)
